src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -150,6 +187,58 @@ def post(
 
         return Response(serialize(team, request.user, TeamWithProjectsSerializer()), status=201)
 
+    def put(
+        self,
+        request: Request,
+        organization: Organization,
+        member: OrganizationMember,
+        team_slug: str,
+    ) -> Response:
+        try:
+            team = Team.objects.get(organization=organization, slug=team_slug)
+        except Team.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        try:
+            omt = OrganizationMemberTeam.objects.get(team=team, organizationmember=member)
+        except OrganizationMemberTeam.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        serializer = OrganizationMemberTeamSerializer(data=request.data, partial=True)
+        if not serializer.is_valid():
+            return Response(status=400)
```

Comment:
Should this include validation errors? Right now there are no clues for the API consumer.